lib/std/Thread.zig

```diff
@@ -75,7 +75,7 @@ pub fn setName(self: Thread, name: []const u8) SetNameError!void {
                 }
             } else {
                 const err = std.c.pthread_setname_np(self.getHandle(), name_with_terminator.ptr);
-                switch (err) {
+                switch (@as(posix.E, @enumFromInt(@as(@typeInfo(posix.E).Enum.tag_type, @intCast(err))))) {
```

Comment:
```suggestion
                switch (@as(posix.E, @enumFromInt(err))) {
```

tfw you're old school and been programming zig since before [0.9.0](https://ziglang.org/download/0.9.0/release-notes.html#intToEnum-Implicitly-Performs-intCast)